modules/osd-create-cluster-ccs.adoc

```diff
@@ -220,6 +220,11 @@ CIDR configurations cannot be changed later. Confirm your selections with your n
 You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
 ====
 +
+.. Provide administrator approval per your cluster update method:
```

Comment:
IBM style guide: Do not use per to mean according to. 
s/per/based on (or something similar)

Reply:
Updated.